core/inject.js

```diff
@@ -352,6 +346,7 @@ Blockly.init_ = function(mainWorkspace) {
       function() {
         Blockly.hideChaff(true);
         Blockly.svgResize(mainWorkspace);
+        Blockly.bumpTopObjectsIntoBounds(mainWorkspace);
```

Comment:
Do you know if there are any performance implications with this change?

Reply:
I don't expect any performance regressions as part of this change because `bumpTopObjectsIntoBounds` checks `hasScrollEdges` and returns early before any metrics calls for workspace with scrollbar pairs.